src/main/java/org/mamute/controllers/QuestionController.java

```diff
@@ -207,6 +211,42 @@ public void showVoteInformation (@Load Question question, String sluggedTitle){
 		redirectToRightUrl(question, sluggedTitle);
 	}
 
+	@Delete
+	public void deleteQuestion(@Load Question question) {
+		if (!environment.supports("deletable.questions")) {
+			result.notFound();
+			return;
+		}
+		if (!currentUser.isModerator() && !question.hasAuthor(currentUser.getCurrent())) {
+			result.use(http()).sendError(403);
+			return;
+		}
```

Comment:
Not exactly the same in this case. Here we should allow only moderators or the question author. If I add this rule, the authors won't be able to remove their questions.